COLLABORATORS.md

```diff
@@ -31,3 +31,4 @@
 - [yous](https://github.com/yous)
 - [marocchino](https://github.com/marocchino)
 - [MJayJang](https://github.com/MJayJang)
+- [ginopc](https://github.com/ginopc)
```

Comment:
As far as I know, collaborators refer to someone who already have some contributions to the project and have commit access to the repository.

Reply:
So I will collaborator after my pull request is approved?

Reply:
Well, once your pull request is approved you become a contributor to the repository. And you will appear into [contributors list](https://github.com/workshopper/how-to-npm/graphs/contributors)

I'm not totally sure what criteria was used to add someone into this file, so I prefer left just original maintainers to modify for now.